code/game/objects/items/_item_reagents.dm

```diff
@@ -64,12 +64,13 @@
 		return TRUE
 
 	var/had_liquids = length(reagents.liquid_volumes)
-	var/trans = reagents.trans_to(target, amount)
+	var/had_units   = target.reagents?.total_volume
+	reagents.trans_to(target, amount)
 
 	if(had_liquids)
 		playsound(src, 'sound/effects/pour.ogg', 25, 1)
 	else
 		// Sounds more like pouring small pellets or dust.
 		playsound(src, 'sound/effects/refill.ogg', 25, 1)
-	to_chat(user, SPAN_NOTICE("You transfer [trans] unit\s of the solution to \the [target].  \The [src] now contains [src.reagents.total_volume] units."))
+	to_chat(user, SPAN_NOTICE("You transfer [target.reagents?.total_volume - had_units] unit\s of the solution to \the [target]. \The [src] now contains [reagents.total_volume] unit\s."))
```

Comment:
why's this necessary? shouldn't we be using the return value of `reagents.trans_to()`?

Reply:
yeah i think it's one of the only ones that doesn't. i really should do another opendream typemaker pass to catch these things tbh. trans_to_turf and all the other trans_to helpers should return whatever `trans_to_holder` does